tests/testthat/test-rstudio_markers.R

```diff
@@ -1,4 +1,3 @@
-context("rstudio_source_markers")
```

Comment:
Why has the test context been removed?

Reply:
See https://www.tidyverse.org/blog/2020/10/testthat-3-0-0/ -- it's already kind of vestigial, and will be deprecated from testthat 3.0.0

Reply:
If you think it's better to keep the removal of `context()` to a separate PR, lmk. just stripped it while I'm in here.

Reply:
Okay, thanks for the link - missed that.
I think it's okay to remove it gradually.